src/providers/awsclient/awsclient.go

```diff
@@ -179,3 +179,40 @@ func (c *awsClient) SendMail(file *domain.File, password string) error {
 	})
 	return err
 }
+
+func (c *awsClient) SendGroupEmails(emails []*domain.Email, sender string) (err error) {
+	svc := ses.New(c.svc, aws.NewConfig().WithRegion(c.sesRegion))
+	errChan := make(chan error)
```

Comment:
better not to use goroutine and also consider adding `sleep` a bit time to avoid API limit of SES
(you can send only several emails per a second)

Reply:
I will add sleep.
Goroutine is no use in this case..